src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -81,8 +93,10 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
         for ($index = $tokens->count() - 1; $index > 0; --$index) {
             $token = $tokens[$index];
 
-            if ($token->equals('(') && !$tokens[$index - 1]->isGivenKind(T_ARRAY)) {
-                $this->fixFunction($tokens, $index);
+            if ($token->equals('(') && !$tokens[$index - 1]->isGivenKind([T_ARRAY, T_WHITESPACE])) {
```

Comment:
Is there a better way to detect a function declaration / function call? If I end up listing all token kinds here this will get hairy.